game/game.go

```diff
@@ -498,6 +567,46 @@ func (g *Game) Save(ctx context.Context) error {
 	return err
 }
 
+func merge(ctx context.Context, r Request, game *Game, user *auth.User) (*Game, error) {
+	games := Games{}
+	gameIDs, err := datastore.NewQuery(gameKind).
+		Filter("Started=", false).
+		Filter("Closed=", false).
+		Filter("Finished=", false).
+		Filter("Private=", false).
+		Filter("NoMerge=", false).
+		Filter("Variant=", game.Variant).
+		Filter("PhaseLengthMinutes=", game.PhaseLengthMinutes).
+		Filter("MaxHated=", game.MaxHated).
+		Filter("MaxHater=", game.MaxHater).
+		Filter("MinRating=", game.MinRating).
+		Filter("MaxRating=", game.MaxRating).
+		Filter("MinReliability=", game.MinReliability).
+		Filter("MinQuickness=", game.MinQuickness).
+		GetAll(ctx, &games)
+	if err != nil {
+		return nil, err
+	}
+	for idx, id := range gameIDs {
+		games[idx].ID = id
+	}
+	sort.Sort(games)
+
+	games.RemoveBanned(ctx, user.Id)
```

Comment:
Was there a test for this? If so I missed it.

Reply:
Oh, right, I didn't. That should be added too.

Reply:
OK, added!